packages/fxa-auth-server/lib/routes/oauth/index.js

```diff
@@ -298,18 +301,34 @@ module.exports = (log, config, oauthdb, db, mailer, devices) => {
             grantType: request.payload.grant_type,
             uid,
             ecosystemAnonId,
-            clientId: request.payload.client_id,
+            service: request.payload.client_id,
```

Comment:
I think we should leave the field name as `clientId` here unless we have a specific reason to change it, if only to minimize any unexpected churn in other graphs that consume this event.